src/node/util.ts

```diff
@@ -170,8 +169,7 @@ export const isHashMatch = async (password: string, hash: string) => {
   try {
     return await argon2.verify(hash, password)
   } catch (error: any) {
-    logger.error(error)
-    return false
+    throw new Error(error)
```

Comment:
Could remove the try/catch since this just re-throws.

Reply:
Ah, you're saying since `argon2` throws? Good point.